x-pack/plugins/cases/server/saved_object_types/migrations/comments.test.ts

```diff
@@ -271,6 +278,55 @@ describe('comments migrations', () => {
         expect((columns[2] as { params: {} }).params).toEqual({ timeZone: 'do not delete' });
       });
     });
+
+    it('should marked lens migrations as deferred correctly', () => {
+      const lensEmbeddableFactory = makeLensEmbeddableFactory(
+        () => ({}),
+        () => ({}),
+        {}
+      );
+
+      const lensMigrations = lensEmbeddableFactory().migrations;
+      const lensMigrationObject =
+        typeof lensMigrations === 'function' ? lensMigrations() : lensMigrations || {};
+      const lensMigrationObjectWithFakeMigration = {
+        ...lensMigrationObject,
+        '8.9.0': jest.fn(),
+        '8.10.0': jest.fn(),
+      };
+
+      const lensVersions = Object.keys(lensMigrationObjectWithFakeMigration);
+      const [lensVersionToBeDeferred, lensVersionToNotBeDeferred] = partition(
+        lensVersions,
+        (version) => gte(version, MIN_DEFERRED_KIBANA_VERSION)
```

Comment:
this is exactly what we do in `isDeferredMigration` if something breaks with `gte` the test will still pass

maybe do

```suggestion
        (version) => ['8.9.0', '8.10.0'].includes(version)
```

`MIN_DEFERRED_KIBANA_VERSION` wont change right?